web/modules/custom/ai_screening_project_track/src/Evaluation.php

```diff
@@ -25,11 +25,30 @@ public static function asOptions(): array {
     ];
   }
 
+  /**
+   * Get evaluation options as lowercase.
+   */
+  public static function asLowerCase(): array {
+    return [
+      self::NONE->value => 'none',
+      self::APPROVED->value => 'approved',
+      self::UNDECIDED->value => 'undecided',
+      self::REFUSED->value => 'refused',
+    ];
+  }
+
   /**
    * Translate a project track evaluation.
    */
   public function getTranslatable(): TranslatableMarkup {
     return static::asOptions()[$this->value];
   }
 
+  /**
+   * Translate a project track evaluation.
+   */
+  public function getAsLowerCase(): string {
+    return static::asLowerCase()[$this->value];
+  }
```

Comment:
I think we should change the enum to be (backed by) a string and use `–>value` instead of `->getAsLowerCase()`.